plugins.py

```diff
@@ -18,8 +20,19 @@ def copyIfExists(plugin):
         print("Copying plugin: " + plugin)
         copy(pluginPath, pluginsVolume)
 
+def copyThemes():
+    copy_tree('./themes', themesPath)
+    print("Copied themes files:")
+    files = []
+    for path, _, folder in os.walk(themesPath):
+        for file in folder:
+            files.append(os.path.join(path, file))
+    for f in files:
+        print(f)
```

Comment:
It looks like you want to print out all copied files. `copy_tree` has a `verbose` flag, maybe that already does the job? (https://docs.python.org/3/distutils/apiref.html#distutils.dir_util.copy_tree)